build_scripts/build_package.sh

```diff
@@ -48,8 +40,10 @@ echo_help() {
   echo "                               will have no effect."
   echo "                               (Default: $DEFAULT_REPO_MONKEY_HOME)"
   echo ""
-  echo "--version                      A version number for the AppImage package."
+  echo "--version                      A version number for the package."
   echo "                               (Default: dev)"
+  echo ""
+  echo "--package                      Which package to build (\"appimage\" or \"docker.\""
```

Comment:
Why not have 2 scripts `build_appimage.sh` and `build_docker.sh`? This would make it easier to build. Now you pretty much need to check docs 100% of the time to remember the convention of keyword and argument.

Reply:
These scripts could be as simple as:
```
./build_package.sh --package appimage $@
```

And that would be an improvement.